app/workers/tasks.php

```diff
@@ -17,6 +17,10 @@
 
 class TasksV1 extends Worker
 {
+    public function getName(): string {
+        return "tasks";
```

Comment:
I think names of all the workers should be in uppercase

Reply:
It will be visible to users in Sentry/RayGun. Do we still want it uppercased? Or lowercase it before pushing? What exactly is the reason behind wanting this uppercase?

Reply:
Okay I was under the impression that other appwrite workers use Title case names. 
But upon checking, they use lower case as well. 

So you can ignore my comment 🙂